test/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+include("RetrieveGoogleTest")
+include_directories (../src)
+find_package( Boost REQUIRED COMPONENTS filesystem )
+
+## Add tests below
+add_subdirectory(test_dictionary)
+add_subdirectory(test_runtime)
```

Comment:
New test suites can be added easily by **add_subdirectory**

* Each sub-directory is a self contained test suite.